docs/index-operations.asciidoc

```diff
@@ -240,16 +240,16 @@ The Get Mappings API will return the mapping details about your indexes and type
 $response = $client->indices()->getMapping();
 
 // Get mappings for all types in 'my_index'
-$params = ['index' => 'my_index'];
+$params = [ 'index' => 'my_index'];
 $response = $client->indices()->getMapping($params);
 
 // Get mappings for all types of 'my_type', regardless of index
-$params = ['type' => 'my_type' ];
+$params = [ 'type' => 'my_type' ];
```

Comment:
I don't think there is a convention, but I'd prefer it to at least be consistent within the page.  E.g. either spaces before/after, or no spaces at all.

I think I'd lean towards no spaces before/after personally.